docs/docs/how-tos/domain-registry.md

```diff
@@ -6,12 +6,13 @@ description: Setting a custom DNS Domain Registry
 
 # Nebari domain registry setup
 
-This domain will be where your application will be exposed.
+Take IP Address generated for the ingress from the terminal and set up the DNS. Setting a DNS record heavily depends on the provider. Below is the links of detailed docs of DNS providers on how to create a DNS record on all the providers out there. 
```

Comment:
### Cloudflare

@asmijafar20 please copy over the docs in this section https://docs.qhub.dev/en/stable/source/installation/setup.html?highlight=cloudflare#cloudflare
and add a comment regarding setting up `--dns-provider=cloudflare` at the end


### Using other DNS providers

Currently, QHub only supports CloudFlare for automatic DNS registration. If an alternate DNS provider is desired, change the --dns-provider flag from Cloudflare to none on the qhub deploy command.

Below are the links to detailed documentation on how to create and manage DNS records on a few providers: